src/python/pants/notes/2.0.x.rst

```diff
@@ -0,0 +1,175 @@
+
+* Remove v1 Python Grpcio codegen backend (#10108)
+  `PR #10108 <https://github.com/pantsbuild/pants/pull/10108>`_
+
+* Remove deprecated `--v2-ui` in favor of `--dynamic-ui` (#10105)
+  `PR #10105 <https://github.com/pantsbuild/pants/pull/10105>`_
+
+* Remove deprecated `--enable-pantsd` in favor of `--pantsd` (#10104)
+  `PR #10104 <https://github.com/pantsbuild/pants/pull/10104>`_
+
+* Remove deprecated `pytest-coverage` scope in favor of `coverage-py` (#10103)
+  `PR #10103 <https://github.com/pantsbuild/pants/pull/10103>`_
+
+* Remove `pants.contrib.confluence` (#10106)
+  `PR #10106 <https://github.com/pantsbuild/pants/pull/10106>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Add support for MyPy to Pants v2 (#10132)
+  `Issue #10131 <https://github.com/pantsbuild/pants/issues/10131>`_
+  `PR #10132 <https://github.com/pantsbuild/pants/pull/10132>`_
+
+* Changes to support Django. (#10088)
+  `PR #10088 <https://github.com/pantsbuild/pants/pull/10088>`_
+
+* Add support for enum list options. (#10083)
+  `Issue #9968 <https://github.com/pantsbuild/pants/issues/9968>`_
+  `PR #10083 <https://github.com/pantsbuild/pants/pull/10083>`_
+
+Bugfixes
+~~~~~~~~
+
+* Do not render pytest at info by default, but give it a description. (#10164)
+  `PR #10164 <https://github.com/pantsbuild/pants/pull/10164>`_
+
+* Preserve source roots in MyPy and Pylint output (#10159)
+  `PR #10159 <https://github.com/pantsbuild/pants/pull/10159>`_
+
+* Set dynamic-ui default based on CI (#10140)
+  `PR #10140 <https://github.com/pantsbuild/pants/pull/10140>`_
+
+* Fix Pytest XML reports and Coverage breaking with remote execution (#10136)
+  `PR #10136 <https://github.com/pantsbuild/pants/pull/10136>`_
+
+* Retry for filesystem changes more quickly, indefinitely, and with logging (#10139)
+  `PR #10139 <https://github.com/pantsbuild/pants/pull/10139>`_
+
+* Normalize paths before determining whether we can watch them. (#10130)
+  `PR #10130 <https://github.com/pantsbuild/pants/pull/10130>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Remove use of importlib for native_engine loading (#10161)
+  `PR #10161 <https://github.com/pantsbuild/pants/pull/10161>`_
+
+* Update sha2 crate dependency (#10163)
+  `PR #10163 <https://github.com/pantsbuild/pants/pull/10163>`_
+
+* Remove unused rust deps. (#10162)
+  `PR #10162 <https://github.com/pantsbuild/pants/pull/10162>`_
+
+* Add message field on workunits and use it to replace some explicit logging (#10158)
+  `PR #10158 <https://github.com/pantsbuild/pants/pull/10158>`_
+
+* Do not include resource targets in v2 MyPy chroot (#10154)
+  `PR #10154 <https://github.com/pantsbuild/pants/pull/10154>`_
+
+* Expose the types consumed to produce a type in a RuleGraph. (#10153)
+  `PR #10153 <https://github.com/pantsbuild/pants/pull/10153>`_
+
+* Upgrade setuptools from 44.0.0 to 47.3.1 (#10149)
+  `PR #10149 <https://github.com/pantsbuild/pants/pull/10149>`_
```

Comment:
Technically an API change

Reply:
Maybe, but I don't think we mix setuptools into anyone's code? Not sure how it would be exposed.

Reply:
`setup-py2`. It means that it won't work with Python 2 by default anymore.